VRDR/DeathRecord.cs

```diff
@@ -40,15 +40,15 @@ public class DeathRecord : VRDRBundle
         private Patient Decedent;
 
         /// <summary>The Decedent's Race and Ethnicity provided by Jurisdiction.</summary>
-        private Observation InputRaceandEthnicity;
+        private Observation InputRaceAndEthnicityObs;
 
-        /// <summary>The Pronouncer of death.</summary>
+        // ///<summary>The Pronouncer of death.</summary>
         //private Practitioner Pronouncer;
```

Comment:
Can we remove the unused code rather than commenting out the comments?

Reply:
Eventually; the goal in this pull request is just to get the warnings to stop